package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "cloudfour.com-patterns",
```

Comment:
Note: Before we publish, should we namespace this for our org? `@cloudfour/patterns` or something like that?

Just a thought, not a blocker for this PR.

Reply:
I thought about that — I could go either way. I didn't change it because I thought `@cloudfour/patterns` (which just says to me "patterns owned by the cloudfour org") was less specific than `cloudfour.com-patterns` (which says to me "patterns specifically for cloudfour.com, and implies ownership by the cloudfour org).

That said, I'm not feeling particularly strongly about this, so I'm happy to change it. Anyone have strong feelings?

Reply:
What about `@cloudfour/cloudfour.com-patterns`. 

That, in theory, would allow to have others in the future that follow the same pattern:
- `@cloudfour/site.com-patterns`
- `@cloudfour/other-site.com-patterns`

Too much? 🤔¯\\\_(ツ)_/¯

Reply:
I like that you're thinking towards the future — I feels repetitive to me on the face, but thinking about adding other sites using that naming pattern it makes sense. I'm down for that suggestion. @tylersticka?

Reply:
Or, a different twist... 
- `@cloudfour-patterns/cloudfour.com`
- `@cloudfour-patterns/site.com`
- `@cloudfour-patterns/other-site.com`

Reply:
@tylersticka settled this on Slack when he pointed out that these patterns are mostly intended for cloudfour.com but may also be used for PWA Stats, our handbook, etc.

Changing the package name to `@cloudfour/patterns`